### PR TITLE
uk_ua.lang

### PR DESCRIPTION
**Pull request by SwEikil**

### CODE REVIEW

**Comment by thexaero** [comment]
Thank you so much!
I'll make sure to review it when I have time.

File: uk_ua.lang [hunk 1]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_radar_over_map_always=Завжди
+gui.xaero_compass_over_everything=Відображати компас над усім
+gui.xaero_box_compass_over_everything=Відображати літери компаса (Пв, Сх, Пд, Зх) над усіма елементами мапи, а не під деякими, наприклад маркери.
+gui.xaero_waypoint_server_shared=Сервер
+gui.xaero_waypoint_somebody_shared=Хтось
+gui.xaero_pac_players=Відобразити група/союзники
+gui.xaero_box_pac_players=Відображати гравців, місцезнаходження яких вам відоме (за допомогою мода "Open Parties and Claims"), коли вони виходять за межі рамки мінімапи або їх не видно з іншої причини.
+gui.xaero_toggle_pac_players=Перемикання група/союзники
+gui.xaero_toggle_pac_chunk_claims=Перемикнути відображення претензій до чанків
+gui.xaero_pac_marked_for_forceload= (Прискорене завантаження)
```
Comment (thexaero) [inline uk_ua.lang:507]: This does not seem correct. Accelerated loading?

File: uk_ua.lang [hunk 2]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_pac_claim_tooltip=%1$s Заява%2$s
+gui.xaero_pac_server_claim_tooltip=Заява сервера%1$s
+gui.xaero_pac_expired_claim_tooltip=Заява сплинула%1$s
+gui.xaero_pac_claims=Відбивання претензій до чанків
```
Comment (thexaero) [inline uk_ua.lang:511]: Seems very wrong. Do you know what the claim feature is?

File: uk_ua.lang [hunk 3]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_pac_claims_border_opacity=Непрозорість межі претензій до чанків.
+gui.xaero_box_pac_claims_border_opacity=Рівень прозорості межі кольорового накладання претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_claims_fill_opacity=Непрозорість заповнення претензій до чанків.
+gui.xaero_box_pac_claims_fill_opacity=Рівень прозорості кольорового заповнення претензій до чанків. Для відображення претензій до чанків необхідно встановити мод Open Parties and Claims.
+gui.xaero_pac_current_claim=Відображати поточне володіння
+gui.xaero_box_pac_current_claim=Відображати інформацію про володіння у чанку, в якому ви перебуваєте. Володіння чанками потребує Open Parties and Claims мод.
+gui.xaero_category_include_list_include_in_super=Включити до супер-категорії
+gui.xaero_box_category_include_list_include_in_super=Автоматично включати всі сутності з цього списку та інших списків, включених до цієї категорії, до суперкатегорії/вищої категорії, окрім сутностей зі списку винятків. Таким чином, вам не доведеться вручну додавати дублікати на кожному рівні.
+gui.xaero_temp_waypoints_global=Тимчасові глобальні маркери
```
Comment (thexaero) [inline uk_ua.lang:521]: This option basically means "should temporary waypoints always be global?". Make sure this is correct.

File: uk_ua.lang [hunk 4]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_box_temp_waypoints_global=Коли увімкнено, тимчасові маркери розглядаються як глобальні та ігнорують обмеження на їх відображення.
+gui.xaero_keep_enlarged_minimap_unlocked=Зберігати напрямок при збільшення
```
Comment (thexaero) [inline uk_ua.lang:523]: Does not seem correct. Should be "keep rotation" not "keep direction". The minimap will continue to rotate, not keep a specific angle.

File: uk_ua.lang [hunk 5]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_box_keep_enlarged_minimap_unlocked=Якщо можливо, не закріплюйте мінімапу на північ при збільшенні, що можливо лише при використанні рамки у формі кола на момент написання цього опису.
+gui.xaero_enlarged_minimap_a_toggle=Збільшення мінімапи увімкнено
```
Comment (thexaero) [inline uk_ua.lang:525]: This option means "Should the Enlarge Minimap key work as a toggle instead of having to hold the key?".

File: uk_ua.lang [hunk 6]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_waypoint_visibility_type_world_map_local= Лок. мапа світу
+gui.xaero_waypoint_visibility_type_world_map_global=Глоб. мапа світу
+gui.xaero_box_visibility_type=Тип видимості у грі/на мапі \n \n Локальний - видимий лише в межах максимальної дальності відображення маркера \n Глобальний - завжди видимий \n Лок. мапа світу - не видима, але вважається локальною \n Глобальна мапа світу. - невидимий, але вважається глобальним
+gui.xaero_option_requires_ingame=Ця опція вимагає щоб ви були у грі.
+gui.xaero_alternative_list_players=Альтернатива "Список гравців"
+gui.xaero_display_stained_glass=Від. пофарбованого скла
+gui.xaero_waypoint_onmap_scale=Масштаб маркерів на мапі
+gui.xaero_waypoint_onmap_scale_auto=Авто
+gui.xaero_switch_to_auto_on_death=Авто. маркер при смерті
+gui.xaero_box_switch_to_auto_on_death=Чи перемикати позначення на авто. при створенні точки смерті. Це не впливає на місце створення точки смерті.
```
Comment (thexaero) [inline uk_ua.lang:536]: This is not about where the deathpoint is created. It's the tooltip for the line 535.

File: uk_ua.lang [hunk 7]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_zoom_on_enlarge=Рівень масштабування при збільшенні
+gui.xaero_zoom_on_enlarge_auto=Авто
+gui.xaero_sort_color=Колір
+gui.xaero_weather_display=Показати інфо. про погоду
+gui.xaero_weather_raining=Дощ/Сніг
+gui.xaero_weather_thundering=Гроза
+gui.xaero_dropdown_waypoint_color=Колір маркеру
+gui.xaero_dropdown_waypoint_set=Набір маркерів
+gui.xaero_dropdown_waypoint_container=Маркер Сервера/Світу
+gui.xaero_dropdown_waypoint_world=Маркер Підсвіту/Виміру
```
Comment (thexaero) [inline uk_ua.lang:546]: Make sure these mean "the world/server of a waypoint" and "the sub-world of a waypoint".

File: uk_ua.lang [hunk 8]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_dropdown_transfer_containers1=Від Сервера/Світу
+gui.xaero_dropdown_transfer_containers2=До Сервера/Світу
+gui.xaero_dropdown_selected_narration=Вибрано %1$s.
+gui.xaero_infodisplay_coords=Координати
+gui.xaero_infodisplay_dimension=Розмір
+gui.xaero_infodisplay_biome=Біом
+gui.xaero_infodisplay_weather=Погода
+gui.xaero_infodisplay_light_level=Рівень освітлення
+gui.xaero_infodisplay_time=Ігровий час
+gui.xaero_infodisplay_highlights=Підсвічування інформації (наприклад, претензії)
```
Comment (thexaero) [inline uk_ua.lang:558]: Make sure this means "info about highlights".

File: uk_ua.lang [hunk 9]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_infodisplay_angles=Кути обертання
+gui.xaero_infodisplay_light_overlay_indicator=Статус накладення освітлення
+gui.xaero_infodisplay_manual_cave_mode_indicator=Статус ручного режиму печер
+gui.xaero_infodisplay_custom_sub_world=Власний маркер підсвіту
```
Comment (thexaero) [inline uk_ua.lang:562]: Make sure this means "Custom Waypoint Sub-World" and not "Custom Sub-World Waypoint". The sub-world is custom.

**Review by thexaero — CHANGES_REQUESTED** [review]
Just looked through the whole file. It seems mostly correct, so great job!

There are quite a few places where I found potential issues though. I don't speak Ukrainian, so I'm only guessing based on automated translation. If a line seems correct to you even after my comment or you have additional questions about it, definitely let me know by replying under specific lines. Also, please don't get offended if I pointed something out incorrectly. I'm just trying to make absolutely sure.

Other than that, A LOT of lines seem very long for where they're located. Have you tested the translation in-game? You can do it directly on 1.12.2.

File: uk_ua.lang [hunk 10]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_OK=OK
 gui.xaero_add_edit=Додати/Змінити
-gui.xaero_new_waypoint=Нова точка
-gui.xaero_initial=Початок
-gui.xaero_edit_waypoint=Змінити точку
-gui.xaero_choose_a_preset=Виберіть заданні налаштування.
-gui.xaero_choose_an_interface=Вибрати інтерфейс
-gui.xaero_minimap=Міні-мапа
+gui.xaero_new_waypoint=Новий маркер
+gui.xaero_initial=Початковий
```
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:8]: If you are talking about line 8, then "Початковий" means "initial letter", "beginning of line", "starting point", etc. Did we understand the context you put in that line correctly?

File: uk_ua.lang [hunk 11]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_instant_waypoint=Швидкий маркер
+gui.xaero_entity_depth=Показувати глибину сутностей
 gui.xaero_show_flowers=Показувати квіти
-gui.xaero_waypoint_names=Ім'я точки над відстанню
-gui.xaero_waypoints_distance=Макс. відст. до точки
-gui.xaero_waypoints_distance_min=Мін. відст. до точки
+gui.xaero_waypoint_names=Ім'я маркера над відстанню
+gui.xaero_waypoints_distance=Макс. відстань відображення маркеру
+gui.xaero_teleport_default_command=Команда телепортації за замовчуванням
+gui.xaero_waypoints_distance_min=Мін. відстань відображення маркеру
```
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:134]: I think so

File: uk_ua.lang [hunk 12]
```diff
@@ -1,154 +1,578 @@
...
-gui.xaero_waypoint_opacity_map=Видимість точок на карті
-gui.xaero_hide_world_names=Приховати назви світів/IP
+gui.xaero_subworld_dimension=Підсвіт/Вимір
+gui.xaero_switch_waypoint_set=Змінити набір маркерів
+gui.xaero_waypoint_opacity_ingame=Прозорість маркерів у грі
+gui.xaero_waypoint_opacity_map=Прозорість маркерів на мапі
+gui.xaero_hide_world_names=Сховати Айпі/Назви світів
+gui.xaero_open_slime=Відкрити при натисканні клавіші
+gui.xaero_options=Налаштування
+gui.xaero_make_automatic=Зробити Світ/Сервер Автоматично
```
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:158]: I understood what you meant. In that case, it should say "Зробити світ/сервер автоматичним"

File: uk_ua.lang [hunk 13]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_make_automatic_msg1=Ви впевнені?
+gui.xaero_make_automatic_msg2=Це поміняє місцями всі дані маркерів між вибраним світом/сервером і автоматичним, таким чином імітуючи автоматичне перемикання вибраного світу/серверу. Переконайтеся, що ви знаєте, що робите.
+gui.xaero_make_multi_automatic=Зробити підсвіт автоматичним
+gui.xaero_make_multi_automatic_msg1=Ви впевнені?
+gui.xaero_make_multi_automatic_msg2=Це поміняє місцями всі дані маркерів між вибраним підсвітом і автоматичним, таким чином імітуючи автоматичне перемикання вибраного підсвіту. Переконайтеся, що ви знаєте, що робите.
```
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:163]: Here the context was guessed correctly, so here is the correct translation

File: uk_ua.lang [hunk 14]
```diff
@@ -1,154 +1,578 @@
...
-gui.xaero_cave_zoom=Наближати печери
-gui.xaero_safe_mode_box=Використовувати Java (CPU) еквівалент мода замість OpenGL (GPU). Безпечний режим - це найпростіший шлях при проблемах із модифікацією. Не всі функції модифікації доступні в даному режимі.
-gui.xaero_height_limit=Обмеження точок по висоті
-gui.xaero_world_map=Карта світу
-gui.xaero_use_world_map=Використовувати чанки карти світу
+gui.xaero_cave_zoom=Наближення в режимі печер
+gui.xaero_safe_mode_box=Використовувати еквівалент цього мода на основі Java (CPU) замість стандартного, на основі OpenGL (GPU). Безпечний режим - це план Б на випадок, якщо мод не працює належним чином. Не всі функції працюють у безпечному режимі. Не використовуйте без нагальної потреби.
+gui.xaero_height_limit=Обмеження висоти сутності
+gui.xaero_world_map=Мапа світу
+gui.xaero_use_world_map=Використовувати чанки з Мапи світу
```
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:143]: No, it has to be from a small one. We just missed it😅

File: uk_ua.lang (hunk 8 above)
Comment (SwEikil) [reply uk_ua.lang:558]: Still, it's better to let it be like this. More players will understand what it is about "Підсвічування інформації (основні моменти)"

File: uk_ua.lang (hunk 10 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:8]: Alright, that seems fine, but it's supposed to say "initials" in plural.

File: uk_ua.lang (hunk 12 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:158]: Yeah, that seems correct!

File: uk_ua.lang [hunk 15]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_box_enlarged_minimap_a_toggle=Використовувати клавішу "Збільшити мінімапу" як перемикач замість утримання.
+gui.xaero_waypoint_visibility_type_world_map_local= Лок. мапа світу
+gui.xaero_waypoint_visibility_type_world_map_global=Глоб. мапа світу
+gui.xaero_box_visibility_type=Тип видимості у грі/на мапі \n \n Локальний - видимий лише в межах максимальної дальності відображення маркера \n Глобальний - завжди видимий \n Лок. мапа світу - не видима, але вважається локальною \n Глобальна мапа світу. - невидимий, але вважається глобальним
+gui.xaero_option_requires_ingame=Ця опція вимагає щоб ви були у грі.
+gui.xaero_alternative_list_players=Альтернатива "Список гравців"
+gui.xaero_display_stained_glass=Від. пофарбованого скла
+gui.xaero_waypoint_onmap_scale=Масштаб маркерів на мапі
+gui.xaero_waypoint_onmap_scale_auto=Авто
+gui.xaero_switch_to_auto_on_death=Авто. маркер при смерті
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:535]: This option does not affect whether deathpoints are created.
What it does:
If you have a non-auto sub-world selected, it switches it back to auto on death.

File: uk_ua.lang (hunk 8 above)
Comment (thexaero) [reply uk_ua.lang:558]: How do you understand what this feature does?

File: uk_ua.lang (hunk 9 above)
Comment (thexaero) [reply uk_ua.lang:562]: Does this mean "Custom Sub-World of Waypoints?"
Does `Власний підсвіт маркерів" seem better to you?

File: uk_ua.lang (hunk 10 above)
[the remark this answers is not in the document]
Comment (SwEikil) [reply uk_ua.lang:8]: If it has to be in multiplication, then probably "Ініціали" will still be better

File: uk_ua.lang (hunk 9 above)
Comment (SwEikil) [reply uk_ua.lang:562]: No, what was written in the translation is the most correct translation

File: uk_ua.lang (hunk 13 above)
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:163]: Sounds a bit confusing. I’ll try to make a review today/tomorrow, compare original line, and the translated one.

File: uk_ua.lang (hunk 9 above)
Comment (thexaero) [reply uk_ua.lang:562]: I see. But just in case, please tell me what this line is used for.

**Comment by VladOliinyk** [comment]
If you guys do not mind, I’ll review that on CrowdIn, since I could see the original line. I’ll make pull request after that

**Comment by thexaero** [comment]
@VladOliinyk Thanks for helping. I don't know if I want to have multiple PRs for the same translation though. Could just be a better idea to suggest changes in this PR and we can discuss them.

**Comment by VladOliinyk** [comment]
> @VladOliinyk Thanks for helping. I don't know if I want to have multiple PRs for the same translation though, if that's what you mean. Could just be a better idea to suggest changes in this PR and we can discuss them.

Got ya. Ok them, since I know the authors of translation (we are from the same community tbh), I’ll just give him the translation and he’ll commit changes to his PR 👌🏻

**Comment by thexaero** [comment]
@VladOliinyk Sounds good!

File: uk_ua.lang (hunk 10 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:8]: Yeah, that seems right!

**Review by ttrafford7 — CHANGES_REQUESTED** [review]

File: uk_ua.lang [hunk 16]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_subworld_dimension=Підсвіт/Вимір
+gui.xaero_switch_waypoint_set=Змінити набір маркерів
+gui.xaero_waypoint_opacity_ingame=Прозорість маркерів у грі
+gui.xaero_waypoint_opacity_map=Прозорість маркерів на мапі
+gui.xaero_hide_world_names=Сховати Айпі/Назви світів
+gui.xaero_open_slime=Відкрити при натисканні клавіші
+gui.xaero_options=Налаштування
+gui.xaero_make_automatic=Зробити Світ/Сервер Автоматично
+gui.xaero_make_automatic_msg1=Ви впевнені?
+gui.xaero_make_automatic_msg2=Це поміняє місцями всі дані маркерів між вибраним світом/сервером і автоматичним, таким чином імітуючи автоматичне перемикання вибраного світу/серверу. Переконайтеся, що ви знаєте, що робите.
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:160]: This seems more correct to me when translating back, but I think it misses the word "making" as in "simulate making", not just "simulate". The user asks for a sub-world to be made "auto" and this feature simulates that.

File: uk_ua.lang (hunk 10 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:8]: Seems good!

File: uk_ua.lang (hunk 12 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:158]: Seems good!

File: uk_ua.lang [hunk 17]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_share_msg2=Переконайтеся, що ви не викриваєте секретне місце!
+gui.xaero_roof=Стеля
+gui.xaero_partial=Частково
+gui.xaero_full=Повністю
+gui.xaero_show_light_level=Відображати рівень освітлення
+gui.xaero_up=[Перейти вгору]
+gui.xaero_down=[Перейти вниз]
+gui.xaero_display_all_sets=Перемкнути відображення Всіх наборів маркерів
+gui.xaero_waypoint_name=назва маркера
+gui.xaero_set_name=Встановити назву
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:187]: Correct!

File: uk_ua.lang [hunk 18]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_share=Поділитись
+gui.xaero_share_msg1=Ви впевнені, що хочете поділитися цим маркером з §cУСІМА§f в чаті?
+gui.xaero_share_msg2=Переконайтеся, що ви не викриваєте секретне місце!
+gui.xaero_roof=Стеля
+gui.xaero_partial=Частково
+gui.xaero_full=Повністю
+gui.xaero_show_light_level=Відображати рівень освітлення
+gui.xaero_up=[Перейти вгору]
+gui.xaero_down=[Перейти вниз]
+gui.xaero_display_all_sets=Перемкнути відображення Всіх наборів маркерів
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:185]: The original seems more correct.

File: uk_ua.lang [hunk 19]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_waypoint_name=назва маркера
+gui.xaero_set_name=Встановити назву
+gui.xaero_render_layer=Накладення шару відображення
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:188]: Seems correct!

File: uk_ua.lang [hunk 20]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_cross_tp=Міжвимірне ТП
+gui.xaero_unreachable_dimension=Недосяжний вимір призначення.
+gui.xaero_show_time=Відображати ігровий час
+gui.xaero_24h=24г
+gui.xaero_12h=12г
+gui.xaero_temporary=тимчасово
+gui.xaero_temporary2=Тимчасово
+gui.xaero_restore=Відновити
+gui.xaero_biomes_vanilla=Біоми в режимі ванільного кольору
+gui.xaero_use_multiworld=Виявлення інших світів
```
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:199]: Yep, sounds right.

**Comment by thexaero** [comment]
@ttrafford7 Thanks for the contribution! 
To test, you can use a .lang to .json converter online. I think this works: https://www.tterrag.com/lang2json/

File: uk_ua.lang (hunk 18 above)
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:185]: `gui.xaero_display_all_sets=Перемкнути відображення всіх наборів маркерів`
looks more correct

File: uk_ua.lang (hunk 16 above)
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:160]: Original is better, as for me:
`gui.xaero_make_automatic_msg2=Це поміняє місцями всі дані маркерів між вибраним світом/сервером і автоматичним, таким чином імітуючи автоматичне перемикання вибраного світу/серверу. Переконайтеся, що ви знаєте, що робите.`

File: uk_ua.lang (hunk 8 above)
Comment (VladOliinyk) [reply uk_ua.lang:558]: I do not understand what "Claims" could mean in the minimap context, but still, the main phrase could be translated a little better:

`gui.xaero_infodisplay_highlights=Виділення інформації (наприклад, претензії)`

highlight = mark something to make it visually more visible

File: uk_ua.lang (hunk 7 above)
Comment (VladOliinyk) [reply uk_ua.lang:546]: There is confusion about the subject in the sentence. 
Here, I guess, we are speaking about dropdown field names, and in that dropdown, we are going to select the World or Dimension for a specific marker. 

So it should be like this:
```
gui.xaero_dropdown_waypoint_container=Сервер/Світ маркеру
gui.xaero_dropdown_waypoint_world=Підсвіт/Вимір маркеру
```

File: uk_ua.lang (hunk 6 above)
Comment (VladOliinyk) [reply uk_ua.lang:536]: Do not understand the context here at all. Neither on eng nor ukr versions.

File: uk_ua.lang (hunk 15 above)
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:535]: What does it mean "non-auto sub-world" ? What is that feature?

File: uk_ua.lang (hunk 3 above)
Comment (VladOliinyk) [reply uk_ua.lang:521]: `gui.xaero_temp_waypoints_global=Глобальні тимчасові маркери` i guess?? 
Or, longer version: 
`gui.xaero_temp_waypoints_global=Відображати тимчасові маркери глобально`
idk, tbh

File: uk_ua.lang (hunk 2 above)
Comment (VladOliinyk) [reply uk_ua.lang:511]: Well, i guess I got the point of what that "claims" are. It's a land claims from plugin/whatever. Okaaaay

File: uk_ua.lang (hunk 1 above)
Comment (VladOliinyk) [reply uk_ua.lang:507]: `gui.xaero_pac_marked_for_forceload= (Примусово завантажений)`

Comment (VladOliinyk) [reply uk_ua.lang:507]: uldated below actually

File: uk_ua.lang [hunk 21]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_entity_category_hostile_tamed=Приручені
+gui.xaero_entity_category_friendly_tamed=Приручені
+gui.xaero_entity_category_players=Гравці
+gui.xaero_entity_category_same_team=З одної команди
+gui.xaero_entity_category_other_teams=З інших команд
+gui.xaero_entity_category_items=Предмети
+gui.xaero_entity_category_other_entities=Інше
+gui.xaero_category_settings=Налаштування
+gui.xaero_category_include_list=Перелік включень
+gui.xaero_box_category_include_list2=Перелік типів сутностей, які будуть додані до (під)категорії поверх інших сутностей, доданих за допомогою параметра "Авто включення".
```
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:403]:
```
gui.xaero_category_hard_include=Примусове включення
gui.xaero_box_category_hard_include2=Контролює, які сутності автоматично включаються до (під)-категорії без необхідності вносити їх до списку включень вручну.

```

File: uk_ua.lang [hunk 22]
```diff
@@ -1,154 +1,578 @@
...
+gui.xaero_toggle_light_overlay=Перемкнути накладання світла
+gui.xaero_box_light_overlay_type=Використання цього накладання може негативно вплинути на продуктивність гри, оскільки наразі воно не використовує фрагменти мапи світу і збільшує частоту оновлення мінімапи. Не тримайте його постійно увімкненим.
+gui.xaero_day=День
+gui.xaero_tamed=Приручені моби
+gui.xaero_untamed_color=Колір неприручених
+gui.xaero_icons_list=Список гравців
+gui.xaero_icons_always=Завжди
+gui.xaero_icons_off=Без значків
+gui.xaero_box_entity_radar_icons=Відображати значки сутностей замість кольорових точок. \n \n Опція "Список гравців" - відображати тільки при натиснутій клавіші "Список гравців" (за замовчуванням TAB).
+gui.xaero_dot_name_scale=Розмір точки імені
```
[the remark this answers is not in the document]
Comment (VladOliinyk) [reply uk_ua.lang:300]: `gui.xaero_dot_name_scale=Масштаб назви точки`

[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:300]: Yeah, that seems correct.

File: uk_ua.lang (hunk 21 above)
[the remark this answers is not in the document]
Comment (thexaero) [reply uk_ua.lang:403]: Ummm, these are not the correct lines. The thread is about gui.xaero_box_category_include_list2.

File: uk_ua.lang (hunk 2 above)
Comment (thexaero) [reply uk_ua.lang:511]: Please test the actual mod with claims if you want to help translate these. 
%2$s is the whole variable reference, %s means it's a string and 2$ is the index, which goes inside the %s, creating %2$s.
There are no plurals in gui.xaero_pac_claim_tooltip.

File: uk_ua.lang (hunk 7 above)
Comment (thexaero) [reply uk_ua.lang:546]: Yeah, that seems correct!

File: uk_ua.lang (hunk 2 above)
Comment (VladOliinyk) [reply uk_ua.lang:511]: then
`gui.xaero_pac_claim_tooltip=Ділянка %1$s%2$s`
as "Claim of `playername``(Forceloadable)`"
right?

File: uk_ua.lang (hunk 5 above)
Comment (SwEikil) [reply uk_ua.lang:525]: This line is correct, you can leave it as it is

**Comment by SwEikil** [comment]
[thexaero](https://github.com/thexaero), I have updated the translation. Please check it out.

Changelog:
1. Corrections have been made to the translation.
2. Edited text for correct display in the game.

This is the final version of the translation. Unfortunately, we do not know how to translate some lines correctly. So, there are two solutions:
1. You can just leave the lines you don't like untranslated.
2. Leave the translation as it is, and if someone will find an inconsistency in the translation on confusion, they can just suggest their own versions.

In any case, I will make a post about the completion of this translation after you approve it. If users don't like the translation of a particular phrase/word and suggest a more correct translation, the changes will be made and you will be notified.

File: uk_ua.lang (hunk 2 above)
Comment (thexaero) [reply uk_ua.lang]: Yeah, that would be correct. I think the latest version right now is also fine though.

**Comment by thexaero** [comment]
Alright, thank you so much! Can you please sign the CLA with the CLAassistant?

The vast majority of it seems great now. I'll do like you suggested. Here are the lines that still don't seem correct to me after the changes, so I'll remove their translations: 
```
gui.xaero_make_multi_automatic_msg2
gui.xaero_make_automatic_msg2
gui.xaero_box_pushbox_boss_health
gui.xaero_box_pushbox_potion_effects
gui.xaero_keep_enlarged_minimap_unlocked
gui.xaero_box_category_include_list2
gui.xaero_enlarged_minimap_a_toggle
gui.xaero_switch_to_auto_on_death
gui.xaero_infodisplay_highlights
gui.xaero_switch_to_auto_on_death
gui.xaero_box_switch_to_auto_on_death
gui.xaero_infodisplay_custom_sub_world
```

Once again, thank you so much for your contribution everyone!

**Comment by SwEikil** [comment]
I already signed it) Okay, if any of the players suggest a correct translation, I'll let you know.
Thank you for this wonderful mod!)

**Comment by thexaero** [comment]
Great, you can probably expect this translation to appear in the mod in the near future :)

**Pull request merged**